<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-0.9.312">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Lars Schöbitz">
<meta name="author" content="Prof.&nbsp;Dr.&nbsp;Elizabeth Tilley">

<title>Untitled</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<script src="proposal-eth-ord-scheme-contribute_files/libs/clipboard/clipboard.min.js"></script>
<script src="proposal-eth-ord-scheme-contribute_files/libs/quarto-html/quarto.js"></script>
<script src="proposal-eth-ord-scheme-contribute_files/libs/quarto-html/popper.min.js"></script>
<script src="proposal-eth-ord-scheme-contribute_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="proposal-eth-ord-scheme-contribute_files/libs/quarto-html/anchor.min.js"></script>
<link href="proposal-eth-ord-scheme-contribute_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link id="quarto-text-highlighting-styles" href="proposal-eth-ord-scheme-contribute_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet">
<script src="proposal-eth-ord-scheme-contribute_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="proposal-eth-ord-scheme-contribute_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="proposal-eth-ord-scheme-contribute_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Untitled</h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">Lars Schöbitz </div>

    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Global Health Engineering, ETH Zurich
          </p>
      </div>
      <div class="quarto-title-meta-contents">Prof.&nbsp;Dr.&nbsp;Elizabeth Tilley </div>

    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Global Health Engineering, ETH Zurich
          </p>
      </div>
    </div>



<div class="quarto-title-meta">

      
    
  </div>
  

</header>

<section id="call-info" class="level1">
<h1>Call Info</h1>
<section id="website" class="level2">
<h2 class="anchored" data-anchor-id="website">Website</h2>
<ul>
<li>Measure 1: Calls for Field-Specific Actions</li>
<li>https://ethrat.ch/en/measure-1-calls-for-field-specific-actions/</li>
<li>Contribute projects will help researchers to contribute to existing open communities built around ORD. They will validate their integration with existing infrastructures (including international ones) and focus on their potential to advance the scientific impact of the field as well as their technical quality.</li>
</ul>
</section>
<section id="email" class="level2">
<h2 class="anchored" data-anchor-id="email">Email</h2>
<ul>
<li>Funding for “Contribute projects” aims at:
<ul>
<li>Contributing a <strong>piece of software, data, hardware</strong>, etc., which <strong>simplifies</strong> an established ORD practice, develops its <strong>research potential</strong>, or otherwise improves it.</li>
<li>Curating or otherwise improving available ORD, for example, by updating existing data to <strong>adapt it to the evolution of ORD practices</strong> or by linking different ORD datasets or by collecting and curating existing data before uploading it to an ORD platform.</li>
<li>Providing <strong>community services</strong>, such as the organization of a <strong>workshop</strong> or training, or <strong>coordination</strong> for the development of an ORD standard.</li>
</ul></li>
</ul>
<p>Funding is available for projects of up to 12 months with an overall budget that must be CHF 30’000.</p>
</section>
<section id="application-guidelines-eth-board" class="level2">
<h2 class="anchored" data-anchor-id="application-guidelines-eth-board">Application guidelines (ETH Board)</h2>
<ul>
<li>https://ethrat.ch/wp-content/uploads/2022/10/Application-guidelines_TrackC-Contribute-projects-3rd-call-1.pdf</li>
<li>Preference will be given to projects which engage and are connected to a community</li>
</ul>
</section>
<section id="application-guidelines-eth-zurich" class="level2">
<h2 class="anchored" data-anchor-id="application-guidelines-eth-zurich">Application guidelines (ETH Zurich)</h2>
<ul>
<li>https://ethrat.ch/wp-content/uploads/2022/06/2022-05-00-ETH-ORD-EPFLETHZ-information-to-applicants-Contribute.pdf</li>
<li>For applicants at ETHZ</li>
<li>In this Contribute call, applicants at ETHZ must contribute 25% of the total budget from the resources of their lab/group/unit, either in-cash or in-kind. See the budgeting guidelines below for eligible contributions</li>
<li>This means an additional 22’500 CHF for us and 7’500 CHF in-kind through my salary</li>
</ul>
</section>
<section id="online-submission-platform" class="level2">
<h2 class="anchored" data-anchor-id="online-submission-platform">Online submission platform</h2>
<ul>
<li>https://epflresearchfunds.gomovein.com/locallogin/5d77bf02089cc332e062a3a2/eng</li>
<li>SNSF guidelines CV</li>
<li>Asbtract (max 1500 characters)</li>
<li>5 keywords</li>
</ul>
</section>
<section id="template" class="level2">
<h2 class="anchored" data-anchor-id="template">Template</h2>
</section>
</section>
<section id="project-proposal" class="level1">
<h1>Project Proposal</h1>
<p>Your project proposal must not exceed 6 pages, excluding the bibliography (A4, Arial 11-point font, 1.5 line-spacing, 2-centimeter margins). Please address all points below.</p>
<section id="proposal-full-title-todo" class="level2">
<h2 class="anchored" data-anchor-id="proposal-full-title-todo">Proposal full title (TODO)</h2>
<p><strong>Instructions</strong></p>
<p>Please include a project title (max. 12 words); this title will be used for all correspondence. Proposal Acronym (mandatory)</p>
</section>
<section id="background-and-motivation" class="level2">
<h2 class="anchored" data-anchor-id="background-and-motivation">Background and motivation</h2>
<p><strong>Instructions</strong></p>
<p>Please explain which aim(s) of the ORD “contribute projects” you seek to address:</p>
<ul>
<li><p>Contributing a piece of software, data, hardware, etc., which simplifies an established ORD practice, develops its research potential, or otherwise improves it.</p></li>
<li><p>Curating or otherwise improving available ORD. For example, by updating existing data to adapt it to the evolution of ORD practices. Or by linking different ORD datasets. Or by collecting and curating existing data before uploading it to an ORD platform.</p></li>
<li><p>Providing community services, such as the organization of a workshop or training, or coordination for the development of an ORD standard.</p></li>
</ul>
<section id="unicefwho-joint-monitoring-programme-jmp" class="level3">
<h3 class="anchored" data-anchor-id="unicefwho-joint-monitoring-programme-jmp">UNICEF/WHO Joint Monitoring Programme (JMP)</h3>
<p>The WHO/UNICEF Joint Monitoring Programme for Water Supply, Sanitation and Hygiene (JMP) was established in 1990 to monitor global progress on drinking water, sanitation and hygiene. The JMP produces estimates for a total of 26 indicators related to water, sanitation and hygiene (WASH). Estimates are produced using linear regression models at the country, regional, and the global levels <span class="citation" data-cites="who/unicefjointprogrammeforwatersupplysanitsationandhygienejmp2018jmp">[<a href="#ref-who/unicefjointprogrammeforwatersupplysanitsationandhygienejmp2018jmp" role="doc-biblioref">1</a>]</span>.</p>
</section>
<section id="jmp-modelled-estimates-data-output" class="level3">
<h3 class="anchored" data-anchor-id="jmp-modelled-estimates-data-output">JMP modelled estimates (data output)</h3>
<p>The produced estimates are the only available source of comprehensive and internationally-comparable information on WASH coverage. Data from JMP are also used in reports used to estimate the global burden of disease associated with water and sanitation and to assess the cost-effectiveness, benefit-cost ratio, global expenditure, and investment needs in global drinking water and sanitation <span class="citation" data-cites="bartram2014globala">[<a href="#ref-bartram2014globala" role="doc-biblioref">2</a>]</span>. More recently the data has been used to quantify the service needs for 772 million onsite sanitation facilities <span class="citation" data-cites="greene2021role">[<a href="#ref-greene2021role" role="doc-biblioref">3</a>]</span>.</p>
</section>
<section id="jmp-data-input" class="level3">
<h3 class="anchored" data-anchor-id="jmp-data-input">JMP data input</h3>
<p>Every two years the JMP updates its global databases to incorporate the latest available national data. <span class="citation" data-cites="who/unicefjointprogrammeforwatersupplysanitsationandhygienejmp2021metadata">[<a href="#ref-who/unicefjointprogrammeforwatersupplysanitsationandhygienejmp2021metadata" role="doc-biblioref">4</a>]</span>. The data is collected from various sources, including censuses, household surveys, administrative data, and others (e.g.&nbsp;research). Data formats between these sources are highly heterogeneous, and a significant amount of effort is invested into extracting data from each source and bringing it into a standardized format.</p>
</section>
<section id="jmp-country-files" class="level3">
<h3 class="anchored" data-anchor-id="jmp-country-files">JMP Country Files</h3>
<p>So-called JMP Country Files are used for recording these data inputs in a hidden sheet of a Microsoft Excel workbook. A statistical analysis software package (Stata 14.0) imports the for all the 232 countries, areas, and territories for which data are available, and runs the estimation model. The resulting estimates are exported back to each country file in another hidden sheet, which in turn feeds a sheet where estimates are produced for the full set of indicators. The result is one JMP Country File for each country. While the modeled estimates are accessible through a World File for all countries, the underlying data input are only accessible through individual country files.</p>
</section>
<section id="use-cases" class="level3">
<h3 class="anchored" data-anchor-id="use-cases">Use cases</h3>
<p>The JMP uses this input data specifically to produce indicator estimates for SDG indicators 6.1 and 6.2, but there is great potential for unforeseen use cases (e.g.&nbsp;research, teaching, joining with other data, etc.). These could be enabled by making the data readily accessible in a form that follows FAIR data principles <span class="citation" data-cites="wilkinson2016fair">[<a href="#ref-wilkinson2016fair" role="doc-biblioref"><strong>wilkinson2016fair?</strong></a>]</span>. There are a range of published studies, which have used the input data, but each of their own would have gone through the effort of extracting and combining all data into a single dataframe for analysis. Further, it is difficult to identify publications that have made use of the JMP input database, because there isn’t a single DOI that could be cited, nor a clear license applied to the the data. Providing open-source software and data in an accessible and cite-able way can lead to significant uptake within a domain. A great example is the Bioconductor project, which is an initiative for the collaborative creation of extensible software for computational biology and bioinformatics <span class="citation" data-cites="gentleman2004bioconductor">[<a href="#ref-gentleman2004bioconductor" role="doc-biblioref">5</a>]</span>. First published in 2004, the software packages developed for the R and python environment are being used by 10s of 1000s of researchers worldwide.</p>
</section>
<section id="problem-statement" class="level3">
<h3 class="anchored" data-anchor-id="problem-statement">Problem Statement</h3>
<p>We seek to address this unused potential by contributing software and data packages that address the aims of this call. Using open-source R Statistical Software, we will curate the data carefully and add valuable variables/categories to the existing data that allow for new and unexpected types of analyses. We seek to address the aim of providing a community service by organizing practical workshops and training on the developed software and data packages. This allows for engagement with the published data on variety of levels and provides a highly relevant dataset that also opens the opportunities for people to see “what is behind the curtain”.</p>
<p>In some preliminary work, we have started to develop R scripts that extract the data from the JMP Country Files and combine them into a single dataframe [reference: washr R Package]. The database contains xxxxx data points from xxxxx variety of sources. We have further started to develop R scripts that replicate the regression models of the JMP using documented JMP methods, but quickly realized that using the JMP Stata script for replicating the analysis would be highly advantageous. A research study is ongoing which applies alternative statistical models to the input data.</p>
</section>
</section>
<section id="ord-project-plan" class="level2">
<h2 class="anchored" data-anchor-id="ord-project-plan">ORD project plan</h2>
<ul>
<li><p>Please describe your project design, including how it addresses the aim(s) of the ORD “contribute projects” described above. Please refer to the scientific literature as well as any preliminary, unpublished results that have informed your approach.</p></li>
<li><p>Which specific gap(s) in knowledge will your work address? Why is your approach innovative?</p></li>
<li><p>What questions will the project answer? What specific outcomes will you focus on?</p></li>
<li><p>Which strategy will you use to address the questions you seek to answer?</p></li>
</ul>
<section id="wp1-document-and-publish-r-data-package-jmpinput" class="level3">
<h3 class="anchored" data-anchor-id="wp1-document-and-publish-r-data-package-jmpinput">WP1: Document and publish R data package <code>jmpinput</code></h3>
<p><strong>Goal:</strong> Make data readily accessible to enable researchers and practitioners to utilize powerful data</p>
<p>In our preliminary work, we have already written a set of functions that extract JMP input data from the JMP Country files. While this is helpful for us, it is not yet easily accessible for others. In this work package, we prepare detailed and concise documentation that describes the data and the structure that it is published in. We prepare comprehensive metadata and complete codebooks. Using the universe of packages available in the R universe, we will publish everything as an R data package, including permissive licenses, a DOI, and an attractive public website that allows anyone to learn about the data. While the data is packaged as an R package, we will also ensure to provide guidance on how it could be used by anyone using a different data analysis software. The approach is innovative because it will lead to a standardized use of the input data. The data processing activities itself are published openly, so that each step taken to compile the resulting dataframe can be reviewed and validated.</p>
<p><strong>Activities:</strong></p>
<ol type="1">
<li>Write up documentation that describes the origin of data and structure that it is provided in</li>
<li>Prepare at least two use cases as “vignettes” that can be published together with the data</li>
<li>Publish data, documentation and “vignettes” as R Package using workflows described in [Ref R Packages]</li>
<li>Use packages of the R environment to publish R Package as a public website, including all documentation and vignettes</li>
</ol>
<p><strong>Aims addressed</strong></p>
<p>This work package focuses on aim 1 and 2 of the call. The research potential increases significantly, as data is now readily available for anyone to use it and build on. The currently established ORD practice of manually downloading MS Excel files is simplified. We further curate the data, so that novel ways of analyzing it will be feasible.</p>
<p><strong>Research questions</strong></p>
<ul>
<li>RQ1.1: What is the impact of sharing the data as a data package? Indicator: Number of citations per year</li>
<li>RQ1.2: What use cases beyond using logistic regression for estimates does the WASH community establish?</li>
</ul>
</section>
<section id="wp2-prepare-and-document-r-software-package-jmpmodel" class="level3">
<h3 class="anchored" data-anchor-id="wp2-prepare-and-document-r-software-package-jmpmodel">WP2: Prepare and document R software Package <code>jmpmodel</code></h3>
<p><strong>Goal:</strong> Develop open-source software to support transparent documentation of modelling activities</p>
<p>The JMP documents methods for the calculation of indicators in detail, however, the actual script which produces the estimates remains as closed access. By publishing this script as open source software anyone view, use, modify, and distribute the work for any purpose, enforced by a permissive open-source license (e.g MIT License). Further, results become reproducible by providing all necessary data (through WP1) and the computer code that runs the regression analysis. This significantly increases transparency of the process and could result in an increase of “trustability” for the actual results. Other added advantages are (1) a complete track record of the complete history of the development process, (2) facilitated collaboration and review process where useful changes and thoughtful contributions can be made to develop our project further; (3) publication of validated research and avoidance of misinformation (4) more efficiency in writing papers, thesis and reports; (5) fair credit for the work; (6) ensured continuity of the work <span class="citation" data-cites="community2019turing">[<a href="#ref-community2019turing" role="doc-biblioref"><strong>community2019turing?</strong></a>]</span>.</p>
<p><strong>Activities:</strong></p>
<ol type="1">
<li>Work with the JMP team to transfer the current Stata script into an R script</li>
<li>Prepare a set of R functions that achieve all necessary data manipulation, modelling and visualization steps</li>
<li>Write up complete documentation for developed R functions</li>
<li>Publish R functions as a complete piece of software as an R Package as described in [Ref R Packages]</li>
<li>Use packages of the R environment to publish R Package as a public website, including all documentation and vignettes</li>
</ol>
<p><strong>Aims addressed</strong></p>
<p>As WP1, this work package focuses on aim 1 and 2 of the call.</p>
<p><strong>Research questions</strong></p>
<ul>
<li>RQ2.1: What are the experienced barriers for publishing the code open-source?</li>
<li>RQ2.2: What are the practical difficulties of transferring a Stata script into a set of R scripts that achieve the same results?</li>
</ul>
</section>
<section id="wp3-collaborate-with-the-jmp-to-promote-use-and-maintenance-of-r-packages" class="level3">
<h3 class="anchored" data-anchor-id="wp3-collaborate-with-the-jmp-to-promote-use-and-maintenance-of-r-packages">WP3: Collaborate with the JMP to promote use and maintenance of R packages</h3>
<p><strong>Goal:</strong> Seek active collaboration on package development to increase potential usage and long-term maintenance of data and software package.</p>
<p>Open-source projects live by the community the contributes to them. They can only be successfully maintained long-term if people invest resources and time into the development. While the project team is well connected with the staff at the JMP, there is currently no defined collaboration on preparing data packages and software. While the project can still be implemented without an agreed collaboration and/or without the Stata script, it would highly benefit from active involvement of the JMP team.</p>
<p><strong>Activities:</strong></p>
<ol type="1">
<li>Identify JMP team member who actively contributes to WP1 and WP2</li>
<li>Work with the JMP team to transfer the current STATA script into an R script</li>
<li>Continuously check-in with JMP team member and general JMP team to update on progress and share intermediate products</li>
<li>Host a 4-hour training/workshop with key members of the JMP in Geneva and/or New York to demonstrate the potential use of the developed R Packages for JMP modeling activities</li>
</ol>
<p><strong>Aims addressed</strong></p>
<p>This work packages addresses aim 3 of the call. The project team will actively engage with the JMP to benefit from the established code, but the at the same time offer opportunities for the JMP team to learn how the developed software could be applied for their own benefit. Some team members already use the R Statistical Software, while others may need to be provided with targeted training. As the Global Health Engineering group already is a leader in providing training for Open Science and Reproducible Research with R, we would use our expertise to transfer these competencies.</p>
<p><strong>Research questions</strong></p>
<ul>
<li>RQ3.1:</li>
<li>RQ3.2:</li>
</ul>
</section>
<section id="wp4-dissemination" class="level3">
<h3 class="anchored" data-anchor-id="wp4-dissemination">WP4: Dissemination</h3>
<p><strong>Goal:</strong> Widely share the developed package for increased use and impact</p>
<p>We will use our established networks and communities to disseminate the developed products. The process can tightly fit into our openwashdata project, funded under the ETH ORD Scheme for Explore Projects and starting March 1st, 2023. The packages will provide a neat opportunity to prepare novel teaching material that motivates WASH professionals and at the same time disseminates our products.</p>
<p><strong>Activities:</strong></p>
<ol type="1">
<li>Host a public participatory live coding online workshop for the openwashdata community to showcase the data packages</li>
<li>Publish open source training materials from online workshop for others to use for own teaching activities (e.g.&nbsp;Caetano Canada WASH thingie, University of Leeds, etc.)</li>
<li>Prepare an article for the Journal of Open Source Software</li>
</ol>
<p><strong>Aims addressed</strong></p>
<p>This work packages addresses aim 3 of the call.</p>
<p><strong>Research questions</strong></p>
<ul>
<li>RQ4.1:</li>
<li>RQ4.2:</li>
</ul>
</section>
</section>
<section id="impact" class="level2">
<h2 class="anchored" data-anchor-id="impact">Impact</h2>
<p>Please address these specific points:</p>
<ul>
<li>How sustainable is the proposed project inside the ETH Domain?</li>
<li>To what extent may an existing or a newly formed community (be able to) engage with the ORD practice(s) built-up during the project?</li>
</ul>
<p>Within the ETH Domain we are able to recruit and access exceptional talent that can contribute to the long-term development and sustainability of the project. It is further complements our activities on the openwashdata project, which established a community of WASH professionals who have a shared interest in data. These professionals will learn how to use the same methods and tools that we apply here and therefore be able to carry on the activities beyond the lifecycle of the project itself.</p>
</section>
<section id="work-packages-and-milestones" class="level2">
<h2 class="anchored" data-anchor-id="work-packages-and-milestones">Work Packages and milestones</h2>
<p>The following Table 2 shows a basic gantt chart against the four work packages, including program activities and community engagement of the four defined learner personas. Column “Lead” abbreviations: LS = Lars Schöbitz. SA = Scientific Assisstant.</p>
<p>The following Table 3 is a list of research questions associated with each of the Work Packages and related activities in Table 2. Any publications derived from this program will be published as open access material, following ORD practices and Open Science standards for computational reproducibility and sharing of data and code under FAIR principles.</p>
<p>Table 2: https://docs.google.com/spreadsheets/d/1pvt08daECVK_M-IY3dx1lNUSjcTVy-8miE0GptWAIlc/edit#gid=0</p>
<p>Table 3: https://docs.google.com/spreadsheets/d/1k4eOJcaWGyJDblThgGnxUCvYUnj6qTaqX3Q-0F6WVyQ/edit#gid=0</p>
</section>
</section>
<section id="resources-including-project-costs" class="level1">
<h1>Resources (including project costs)</h1>
<p>Table 4: https://docs.google.com/spreadsheets/d/1jQE1qrO0T88aXjeARYQaQ3sNsX8UaURJdm7B-yAvDN0/edit#gid=0</p>
<section id="bibliography" class="level2 unnumbered">


</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">Bibliography</h2><div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-who/unicefjointprogrammeforwatersupplysanitsationandhygienejmp2018jmp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">WHO/UNICEF Joint Programme for Water Supply, Sanitsation and Hygiene (JMP). <span>JMP Methodology</span> - 2017 <span>Update</span> &amp; <span>SDG Baselines</span>. 2018 Mar. </div>
</div>
<div id="ref-bartram2014globala" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Bartram J, Brocklehurst C, Fisher MB, Luyendijk R, Hossain R, Wardlaw T, Gordon B. <a href="https://doi.org/10.3390/ijerph110808137">Global <span>Monitoring</span> of <span>Water Supply</span> and <span>Sanitation</span>: <span>History</span>, <span>Methods</span> and <span>Future Challenges</span></a>. International Journal of Environmental Research and Public Health. 2014 Aug;11(8):8137–65. </div>
</div>
<div id="ref-greene2021role" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Greene N, Hennessy S, Rogers TW, Tsai J, de los Reyes III FL. <a href="https://doi.org/10.1016/j.jenvman.2021.112612">The role of emptying services in provision of safely managed sanitation: <span>A</span> classification and quantification of the needs of <span>LMICs</span></a>. Journal of Environmental Management. 2021 Jul;290:112612. </div>
</div>
<div id="ref-who/unicefjointprogrammeforwatersupplysanitsationandhygienejmp2021metadata" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">WHO/UNICEF Joint Programme for Water Supply, Sanitsation and Hygiene (JMP). Metadata: <span>SDG</span> global indicator 6.2.1a. 2021 Dec. </div>
</div>
<div id="ref-gentleman2004bioconductor" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, Dudoit S, Ellis B, Gautier L, Ge Y, Gentry J, Hornik K, Hothorn T, Huber W, Iacus S, Irizarry R, Leisch F, Li C, Maechler M, Rossini AJ, Sawitzki G, Smith C, Smyth G, Tierney L, Yang JY, Zhang J. <a href="https://doi.org/10.1186/gb-2004-5-10-r80">Bioconductor: Open software development for computational biology and bioinformatics</a>. Genome Biology. 2004 Sep;5(10):R80. </div>
</div>
</div></section><section class="quarto-appendix-contents"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{schöbitz,
  author = {Lars Schöbitz and Prof. Dr. Elizabeth Tilley},
  title = {Untitled},
  langid = {en}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-schöbitz" class="csl-entry quarto-appendix-citeas" role="doc-biblioentry">
<div class="">Lars
Schöbitz, Prof. Dr. Elizabeth Tilley. Untitled. </div>
</div></div></section></div></main>
<!-- /main column -->
<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>